modules/ROOT/pages/endpoint-https-receive.adoc

```diff
@@ -59,6 +60,17 @@ When you create this API in API Manager:
 NOTE: Using this feature requires an API Manager license.
 |
 
+|HTTPS
+a|Applies to CloudHub deployments.
+
+Indicates whether Partner Manager deploys this endpoint as an HTTPS or HTTP listener:
+
+* If enabled (default), Partner Manager deploys this endpoint as an HTTPS listener.
+* If disabled, Partner Manager deploys this endpoint as an HTTP listener.
+
+When deploying to Anypoint Runtime Fabric, Partner Manager always deploys this endpoint as an HTTP listener. With this type of deployment, you must configure an Ingress controller to provide external consumers with an HTTPS endpoint. For more information, see xref:runtime-fabric::custom-ingress-configuration.adoc[Configuring Ingress for Runtime Fabric].
```

Comment:
Hardcoded link text

Reply:
Not sure if this would work going to another doc